misc-tools/dj_make_chroot.in

```diff
@@ -185,7 +193,13 @@ if [ "$DISTRO" = 'Ubuntu' ]; then
     INCLUDEDEBS=""
 
     # Packages to install after upgrade (space separated):
-    INSTALLDEBS="gcc g++ make default-jdk-headless default-jre-headless pypy3 locales"
+    INSTALLDEBS="gcc g++ make default-jdk-headless default-jre-headless locales"
+    if [ "$ARCH" != "i386" ]; then
```

Comment:
Better to not use negative expressions in tests (since they're harder to parse by humans), so write
```suggestion
    if [ "$ARCH" = "i386" ]; then
```
and swap the lines below.